make/tests

```diff
@@ -208,3 +208,7 @@ endif
 # Include the test dependencies
 ##
 include make/test-math-dependencies
+
+.PHONY: test 
```

Comment:
> I don't think there should be a circular dependency here.

I am happy to comply, but you realize that the purpose of PHONY is to tell make that the rule does not create a file called "test", right? See https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html